go.mod

```diff
@@ -3,11 +3,11 @@ module github.com/ElrondNetwork/elrond-sdk-erdgo
 go 1.14
 
 require (
-	github.com/ElrondNetwork/elrond-go v1.2.36-0.20220131130250-db07b484bee6
-	github.com/ElrondNetwork/elrond-go-core v1.1.7
+	github.com/ElrondNetwork/elrond-go v1.3.6-0.20220221102635-7228fcdd5299
```

Comment:
can be done in another PR: reference the latest commit from elrond-go in order get rid of one arwen v1.4 reference

Reply:
ok :+1: